node/silkworm/stagedsync/stage_execution.cpp

```diff
@@ -84,7 +84,8 @@ StageResult Execution::forward(db::RWTxn& txn) {
         prune_receipts = std::min(prune_receipts, hashstate_stage_progress - 1);
     }
 
-    AnalysisCache analysis_cache;
+    static constexpr size_t kCacheSize{5'000};
+    lru_cache<evmc::bytes32, std::shared_ptr<evmone::baseline::CodeAnalysis>> analysis_cache{kCacheSize};
```

Comment:
I don't get this. Isn't `lru_cache<evmc::bytes32, std::shared_ptr<evmone::baseline::CodeAnalysis>>` the `cache_` member of `AnalysisCache` class ?

Reply:
No, in AdvancedAnalysisCache it's  lru_cache<evmc::bytes32, std::shared_ptr<evmone::**advanced::AdvancedCodeAnalysis**>>

Reply:
I'll introduce a type alias for clarity.